tests/integration/test_integration.py

```diff
@@ -34,6 +34,7 @@
 LOKI_APPLICATION_NAME = "loki-k8s"
 PROMETHEUS_APPLICATION_NAME = "prometheus-k8s"
 SELF_SIGNED_CERTIFICATES_APPLICATION_NAME = "self-signed-certificates"
+SELF_SIGNED_CERTIFICATES_REVISION = 263
```

Comment:
Why pin to a specific revision and not use the actual revision that people will used when deploying Vault (i.e. whatever the latest revision on `1/stable`?

Reply:
To make the point more real -- this was a very real problem with self-signed-certificates and tls-certificates-requirer. It took quite a while to figure out which revisions were compatible with which as I detangled the V4 updates. This required me to write a totally separate version of the integration tests that I was using to test my backports, because the integration tests for v3 of the lib would no longer work with the latest version of the tls-certificates requirer, and vice-versa. This would not have been an issue if the revisions were pinned.

Reply:
I agree we could be more consistent. I'll add a story to the backlog to follow up on this.

Reply:
https://warthogs.atlassian.net/browse/TLSENG-681